### ISSUE TITLE
npm run build not working

### ISSUE THREAD

**Issue by sanjayror** [microsoft/BotFramework-WebChat#3379]
🚨 I'm having issue when npm run build 🚨

npm run build not working in BotFramework-webchat

Node Version : v12.14.1
NPM Version : 6.13.4

![image](https://user-images.githubusercontent.com/15170372/89259098-347f4700-d647-11ea-8316-24dc94bf219d.png)

Complete Log : 
[2020-08-04T06_09_17_616Z-debug.log](https://github.com/microsoft/BotFramework-WebChat/files/5020374/2020-08-04T06_09_17_616Z-debug.log)

**Comment by stevkan** [comment]
Hi @sanjayror, I just cloned and built the repo with no issues. Are you following the steps listed in the build [docs](https://github.com/microsoft/BotFramework-WebChat/blob/master/docs/BUILD_SCRIPTS.md#build-scripts)? As they indicate, from the root directory you'll want to:

- Prep the repo by running
    - `npm ci`
    - `npm install`
- Run `npm run bootstrap`
- Run `npm run build`

**Comment by sanjayror** [comment]
Hi @stevkan 
Thanks for Quick response.

Now I'm able to npm run build

![image](https://user-images.githubusercontent.com/15170372/89368130-62709400-d6f8-11ea-8a50-8855a8b796f1.png)


but after the [npm run build success] bundle from **/packages/bundle/dist/ is empty**
**dist folder is not created**, I checked the complete package but dist folder not found.
please help why my dist folder not created !!

**Comment by stevkan** [comment]
@sanjayror, Look in `/packages/bundle/dist`. You _should_ find the files created there.

**Comment by sanjayror** [comment]
Hi @stevkan 

Thanks for your response 

![image](https://user-images.githubusercontent.com/15170372/89553279-ee82d880-d82a-11ea-935f-4e733c446dc5.png)

If I see /packages/bundle/ there is not **dist** folder.

m'i missing something ?
Can you suggest, that will be very helpful.

**Comment by sanjayror** [comment]
Hi @stevkan 

Thanks for your time.
I need to generate webchat.js only once. If possible can you please generate webchat.js file for me.

Path : 
https://github.com/microsoft/BotFramework-WebChat/blob/master/packages/bundle/src/renderMarkdown.js
After make a little change in this file :-

![image](https://user-images.githubusercontent.com/15170372/89560243-55f15600-d834-11ea-8d3a-6e8f55dccec1.png)


  const html = customMarkdownIt.render(markdown);
  return html;

  //return sanitizeHTML(html, SANITIZE_HTML_OPTIONS);

because I want to skip sanitize html process.
Can you please generate webchat.js for me.

Thanks in advance

**Comment by stevkan** [comment]
Hi @sanjayror, unfortunately, that is not something I can do for you. I can continue to help you resolve your issue, however.

When you run any of the commands to build Web Chat, do you see any errors listed if you scroll back upwards? Also, you might want to clone the repo in your system's root directory just in case there is an issue with the length of some of the directory/filename names.

**Comment by sanjayror** [comment]
Hi @stevkan 

Thanks for help
i'm attaching the npm run build complete log of cmd. Can you please look into this. I can't see error, Is there any permission issue !!

[npm run build.txt](https://github.com/microsoft/BotFramework-WebChat/files/5039170/npm.run.build.txt)

**Comment by stevkan** [comment]
@sanjayror, I noticed in the first log file you provided you are working in your "c:\" drive and in the second file you are working in your "e:\" drive. Why the two and could this be part of the issue?

Also, I am running Node v14.6.0 and npm v6.14.5. Can you try updating and try again, as well?

**Comment by sanjayror** [comment]
@stevkan now I'm able to generate files

Thanks for your support

**Comment by stevkan** [comment]
Hi @sanjayror, I'm glad you were able to get it work. As for your new question on size, I see you opened a new issue, #3408, which is the appropriate action.

Closing this as resolved.
